src/app/notifications/components/Notifications.tsx

```diff
@@ -35,20 +39,36 @@ const connector = connect(
 );
 
 const notificationWidth = 30;
+const getInset = (props: Pick<Props, 'toastInset' | 'inlineBreak'>) => props.toastInset || notificationWidth;
 
-// TODO - magic number to be replaced in `top` when scroll behavior is developed in openstax/unified#179
+const getInlineBreak = (props: Pick<Props, 'toastInset' | 'inlineBreak'>) => {
+  return ((props.inlineBreak || contentWrapperMaxWidth) + notificationWidth * 2) * 10 / 16 + 'em';
+};
+
+const setInlineStyle = (props: Pick<Props, 'toastInset' | 'inlineBreak'>, style: FlattenSimpleInterpolation) => css`
```

Comment:
because its a media query react doesn't re-render when the layout changes, so you can't pass a boolean around for if its the inline display or not. 

and because the media query is dynamic based on the props passed into the Notifications component, you can't simply import it from a constants or theme file.

this `setInlineStyle` could alternatively just be a prop that gets passed down for what the media query width should be, but doing it this way dries up the actual media query code so if we ever change it from `max-` or `min-` width it'll only be in one place.